port/platform/platformio/example/esp32-cell-thingstream-mqttsn.cpp

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2019-2023 u-blox
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ *
+ * A simple demo application showing how to set up
+ * a MQTT-SN connection to thingstream using the ubxlib.
+ * By: Jan-Ole Giebel
+ *
+*/
+#include <Arduino.h>
+#include <string.h>
+#include <stdio.h>
+#include <driver/uart.h> //For using UART_NUM_1, UART_NUM_2 and UART_NUM_*
+
+#include "ubxlib.h"
+
+#define BROKER_NAME "10.7.0.55:2442" // The thingstream MQTT-SN-Broker IP and port.
+
+// Change all -1 values below to appropriate pin and settings values
+// appropriate for your module connection.
+static const uNetworkCfgCell_t gNetworkCfg = {
+    .type = U_NETWORK_TYPE_CELL,
+    .pApn = "tsudp",       // For the thingstream SIM use "tsiot", though for me [Jan-Ole Giebel, based in Germany] this APN worked just fine. Use NULL for default.
```

Comment:
Ah, thanks for the clarification!

I did not know that yet.

I updated the file and changed the name to include `mqttAnywhere` so that it is easier to identify with the u-blox services.